cli/bzlmod/bzlmod.go

```diff
@@ -0,0 +1,70 @@
+package bzlmod
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/buildbuddy-io/buildbuddy/cli/bazelisk"
+)
+
+var (
+	once       sync.Once
+	useModules bool
+	infoErr    error
+)
+
+func UseModules() (bool, error) {
```

Comment:
nit: this can be simplified using the new `sync.OnceValues`

```
var UseModules = sync.OnceValues(func() (bool, error) {
  // return directly instead of setting globals
})
```

Reply:
Good call. Didn't know about this 😅